SwiftUIDZ/SwiftUIDZ/ContentView.swift

```diff
@@ -7,15 +7,156 @@
+        static let buttonWidthNumber: CGFloat = 100
+        static let buttonHeightNumber: CGFloat = 30
+        static let cornerRadiusNumber: CGFloat = 10
+        static let animationResponseNumber = 0.5
+        static let animationDampingFractionNumber = 0.7
+        static let animationBlendDurationNumber = 0.3
+        static let balanceTwoOffsetY: CGFloat = 30
+        static let biasViewNumber: CGFloat = 200
+        static let biasTextNumber: CGFloat = 100
+    }
+    
+    @State var isOnToogle = false
+    @State var balance = Constants.zeroNumber
+    @State var balanceTwo = Constants.fiftyNumber
+    @State var isRefillTheBalanceAlert = false
+    @State var isBalanceToppedUp = false
+    @State var actionSheetText = Constants.emptyString
+    @State var isTransfersMoneyAlert = false
+    @State var isCreateBankAccount = false
+    @State var openTwoBankAccount = Constants.emptyString
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            ZStack {
+                HStack {
+                    VStack {
+                        Text(Constants.walletText)
+                        Spacer().frame(height: Constants.walletTextSpacerHeightNumber)
+                        transfersMoneyButton()
+                        Spacer().frame(height: Constants.transfersMoneyButtonSpacerHeightNumber)
+                        refillTheBalanceButton()
+                        Spacer().frame(height: Constants.refillTheBalanceButtonSpacerHeightNumber)
+                        Text(Constants.profileText)
+                        Spacer().frame(height: Constants.profileTextSpacerHeightNumber)
+                        openBankAccountButton()
+                        Spacer().frame(height: Constants.openBankAccountButtonSpacerHeightNumber)
+                    }
+                    Spacer()
+                }
+                Spacer()
+                RoundedRectangle(cornerRadius: Constants.cornerRadiusNumber).fill(Color.yellow).offset(x: isOnToogle ? Constants.biasViewNumber : Constants.zeroCGFloatNumber)
+                VStack {
+                    makeBankAccount(balance: balance)
+                    
+                    if isCreateBankAccount {
+                        makeBankAccount(balance: balanceTwo).offset(y: Constants.balanceTwoOffsetY)
+                    }
+                }
+                .offset(x: isOnToogle ? Constants.biasTextNumber : Constants.zeroCGFloatNumber)
+            }
+            Toggle(isOn: $isOnToogle, label: { Text(Constants.showSettingsText)
+            }).padding()
+        }.animation(.spring(response: Constants.animationResponseNumber, dampingFraction: Constants.animationDampingFractionNumber, blendDuration: Constants.animationBlendDurationNumber))
+    }
+    
+    fileprivate func makeBankAccount(balance: Int) -> some View {
```

Comment:
```suggestion
    private func makeBankAccountView(balance: Int) -> some View {
```
Всегда окончание View